frontend/src/concepts/connectionTypes/fields/FileFormField.tsx

```diff
@@ -14,31 +17,68 @@ const FileFormField: React.FC<FieldProps<FileField>> = ({
   const isPreview = mode === 'preview';
   const [isLoading, setIsLoading] = React.useState(false);
   const [filename, setFilename] = React.useState('');
+  const [isRejected, setIsRejected] = React.useState(false);
```

Comment:
`isRejected` should be cleared after the user makes any change to the extension.
Scenario:
- Add file extension `.jpg`
- DnD a .png
- Observe error
- Now delete the or change the file extension
- Error remains but has a different value. 
- I think the error should disappear since it's no longer the same.